crates/sui-rpc-api/proto/sui.types.proto

```diff
@@ -0,0 +1,1040 @@
+syntax = "proto3";
+
+package sui.types;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+
+message Address {
+  bytes address = 1;
+}
+
+message ObjectId {
+  bytes object_id = 1;
+}
+
+message Digest {
+  bytes digest = 1;
+}
+
+message Bcs {
+  bytes bcs = 1;
+  // optional bytes name = 2;
+}
+
+// Checkpoint {{{
+
+message CheckpointSummary {
+  // Reserve field 1 and 2 for possible bcs bytes and/or Digest of type
+  reserved 1, 2;
+
+  uint64 epoch = 3;
+  uint64 sequence_number = 4;
+  uint64 total_network_transactions = 5;
+  Digest content_digest = 6;
+  optional Digest previous_digest = 7;
+
+  GasCostSummary epoch_rolling_gas_cost_summary = 8;
+
+  google.protobuf.Timestamp timestamp = 9;
```

Comment:
I was on the fence which how to represent timestamps. the actual sui data type is in milliseconds instead of the standard `google.protobuf.Timestamp` type that is generally recommended. Maybe to ensure parity with the core types we change this to be milliseconds and then use Timestamp on the top-level request types?